SpatialGDK/Source/SpatialGDK/Private/Interop/SpatialPlayerSpawner.cpp

```diff
@@ -61,7 +61,8 @@ void USpatialPlayerSpawner::ReceivePlayerSpawnRequest(Schema_Object* Payload, co
 	// Send a successful response if the player has been accepted, either from this request or one in the past.
 	Worker_CommandResponse CommandResponse = {};
 	CommandResponse.component_id = SpatialConstants::PLAYER_SPAWNER_COMPONENT_ID;
-	CommandResponse.schema_type = Schema_CreateCommandResponse(SpatialConstants::PLAYER_SPAWNER_COMPONENT_ID, 1);
+	CommandResponse.command_index = 1;
```

Comment:
Not sure what this 1 is. Maybe create a SpatialConstants::SOMETHING?

Reply:
It looks like the command index so something like PLAYER_SPAWNER_COMMAND_INDEX or whatever the command is called.

Reply:
The entire class has hardcoded constants in there, for now I will leave it as is, but may be worth looking into in some future refactoring work.